### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by ajinkyap12** [adrianhajdin/project_ai_mern_image_generation#63]
Why is this happing ?

For 1 page:     TypeError: Failed to fetch 
For 2 page :    SyntaxError: Unexpected end of JSON input

**Comment by pushkardangi** [comment]
Getting the same error for page 1

**Comment by ajinkyap12** [comment]
No .. but I think so our OpenAi api is not working

**Comment by pushkardangi** [comment]
Yeah, we need to change the code for api. Because they changed it (OpenAI)

**Comment by ajinkyap12** [comment]
But what should be change
